python/lsst/ip/isr/assembleCcdTask.py

```diff
@@ -30,30 +30,31 @@
 
 __all__ = ["AssembleCcdTask"]
 
+
 class AssembleCcdConfig(pexConfig.Config):
     setGain = pexConfig.Field(
-        doc = "set gain?",
-        dtype = bool,
-        default = True,
+        doc="set gain?",
+        dtype=bool,
+        default=True,
     )
     doRenorm = pexConfig.Field(
-        doc = "renormalize to a gain of 1? (ignored if setGain false). "
-            "Setting to True gives 1 ADU per electron. "
-            "Setting to True is not recommended for mosaic cameras because it breaks normalization across "
+        doc="renormalize to a gain of 1? (ignored if setGain false). "
+        "Setting to True gives 1 ADU per electron. "
+        "Setting to True is not recommended for mosaic cameras because it breaks normalization across "
```

Comment:
Look at the line below this one. Different indenting. I think I prefer the 4 character indent...